thunder/dynamo/report.py

```diff
@@ -196,6 +197,14 @@ def __init__(self, graph: torch.fx.GraphModule, graph_name: str, example_input_m
         self.graph = graph
         self.graph_name = graph_name
         self.example_input_meta = example_input_meta
+        self.ops = [node.target for node in self.graph.graph.nodes if node.op in ("call_function", "call_method")]
+
+    def __str__(self):
+        output = f"Graph Name: {self.graph_name}\n"
+        output += "  Operators used in the graph:\n"
```

Comment:
Inserting spaces is fine, but in the future it might be clearer to put tabs in (`\t`)